src_cam/radiation.F90

```diff
@@ -1398,29 +1370,19 @@ subroutine radiation_tend( &
 
             if (active_calls(icall)) then
 
+               ! OSLO_AERO BEGIN
                ! update the concentrations in the RRTMG state object
                call  rrtmg_state_update( state, pbuf, icall, r_state)
 
-               call rad_rrtmg_lw(                                             &
-                    lchnk, ncol, num_rrtmg_levs, r_state, state%pmid,         &
-                    aer_lw_abs*0.0_r8, cldfprime, c_cld_lw_abs, qrl, rd%qrlc, &
-                    flns, flnt, rd%flnsc, rd%flntc, cam_out%flwds,            &
-                    rd%flut, rd%flutc, fnl, fcnl, rd%fldsc,                   &
-                    lu, ld)
-
-               call outfld('FLNT_DRF',flnt(:)    , pcols, lchnk)
-               call outfld('FLNTCDRF',rd%flntc(:), pcols, lchnk)
+               ! Note that aer_lw_abs is calculated in the call to oslo_aero_optical_params_calc
```

Comment:
Does that replace the call to `rad_rrtmg_lw`? What about the `outfld` calls on lines 1411 and 1412?

Reply:
@gold2718 - Can you please clarify what you mean by this?